### PR TITLE
feat: P4PU-146 implement business logic to retrieve transactions list

### PR DESCRIPTION
**Pull request by Giuseppe-LaManna**
#### List of Changes
- Added business logic to retrieve transactions list 
- removed continuation-token field
- Added Mapper
- Added Faker
- Added Junit

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/java/it/gov/pagopa/arc/service/TransactionsServiceImpl.java
Comment (and-mora) [inline src/main/java/it/gov/pagopa/arc/service/TransactionsServiceImpl.java:1]: DTO mapping should not happen in the service for the sake of layer isolation

File: src/main/java/it/gov/pagopa/arc/utils/Utilities.java
Comment (and-mora) [inline src/main/java/it/gov/pagopa/arc/utils/Utilities.java:1]: Questionable way to handle exceptions

File: src/test/java/it/gov/pagopa/arc/utils/UtilitiesTest.java
Comment (and-mora) [inline src/test/java/it/gov/pagopa/arc/utils/UtilitiesTest.java:1]: need to test the edge cases

File: src/main/java/it/gov/pagopa/arc/utils/Utilities.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            }else {
+                return null;
+            }
+        } else{
+            return null;
+        }
+    }
+
+    /** To convert euro from String to double */
+    public static Double euroStringToLong(String euroString){
```
Comment (and-mora) [inline src/main/java/it/gov/pagopa/arc/utils/Utilities.java:31]: reduce visibility scope if not used externally

**Review by and-mora — CHANGES_REQUESTED** [review]

File: src/main/java/it/gov/pagopa/arc/service/TransactionsServiceImpl.java [hunk 2]
```diff
@@ -1,47 +1,22 @@
...
+        this.bizEventsService = bizEventsService;
     }
 
     @Override
-    public List<TransactionDTO> retrieveTransactionsList(String fiscalCode) {
-        List<TransactionDTO> transactionDTOList = new ArrayList<>();
-
-        if (StringUtils.isNotBlank(fiscalCode)) {
-            try {
-                InputStream mockedTransactionsListJson = TransactionDTO.class.getResourceAsStream("/mock/transactions_response.json");
```
Comment (and-mora) [inline src/main/java/it/gov/pagopa/arc/service/TransactionsServiceImpl.java:31]: have you removed the file referred by?

**Review by and-mora — APPROVED** [review]
